src/ui/verificationSymbolStatusView.ts

```diff
@@ -186,7 +186,7 @@ export default class VerificationSymbolStatusView {
       });
     }
     collectTestItems(controller.items);
-    this.getVerifiableRangesPromise(params.uri).resolve(allTestItems.map(v => v.range!));
+    this.getVerifiableRangesPromise(Uri.parse(params.uri).toString()).resolve(allTestItems.map(v => v.range!));
```

Comment:
I would prefer it if all variables named `uriString` already have the right encoding. This seems like too late of a place to fix the encoding.

Maybe we can do

```
  private async update(params: IVerificationSymbolStatusParams): Promise<void> {
    await this.noRunCreationInProgress;
    const uri = Uri.parse(params.uri);
    params.uri = uri.toString(); // <--- new line
```

that might also resolve other issues where uriString comparison is involved, like tab switching.